xml/System.Data/DataRowView.xml

```diff
@@ -342,10 +347,11 @@
       </Parameters>
       <Docs>
         <param name="relationName">A string containing the <see cref="T:System.Data.DataRelation" /> name.</param>
-        <param name="followParent">The parent</param>
+        <param name="followParent">The parent object.</param>
```

Comment:
Thanks @divega. Would this description sound better?:

```suggestion
        <param name="followParent"><see langword="true" /> if the created child view should be kept in sync with the parent; <see langword="false" /> otherwise.</param>
```

Reply:
yes, looks good.